airflow/www/views.py

```diff
@@ -2408,6 +2408,55 @@ def prefill_form(self, form, pk):
                 field.data = value
 
 
+class PluginView(AirflowBaseView):
+    """View to show Airflow Plugins"""
+    default_view = 'list'
+
+    plugins_attributes_to_dump = [
+        "operators",
+        "sensors",
+        "hooks",
+        "executors",
+        "macros",
+        "admin_views",
+        "flask_blueprints",
+        "menu_links",
+        "appbuilder_views",
+        "appbuilder_menu_items",
+        "global_operator_extra_links",
+        "operator_extra_links",
+    ]
+
+    @expose('/plugin')
+    @has_access
+    def list(self):
+        plugins_manager.ensure_plugins_loaded()
+        plugins_manager.integrate_dag_plugins()
+        plugins_manager.integrate_executor_plugins()
+        plugins_manager.initialize_extra_operators_links_plugins()
+        plugins_manager.initialize_web_ui_plugins()
+
+        plugins = []
+        for plugin_no, plugin in enumerate(plugins_manager.plugins, 1):
+            plugin_data = {}
+            plugin_data['plugin_no'] = plugin_no
+            plugin_data['plugin_name'] = plugin.name
+            plugin_data['attrs'] = {}
```

Comment:
Can you add warning when plugin may not be compatible with the current Airflow version?
https://github.com/apache/airflow/blob/master/airflow/plugins_manager.py#L274

Reply:
@mik-laj The warning will already come in the `plugins_manager.initialize_web_ui_plugins()` step, right? https://github.com/apache/airflow/pull/10770/files#diff-948e87b4f8f644b3ad8c7950958df033R2437